zamba/models/cnnensemble/src/config.py

```diff
@@ -1,6 +1,11 @@
+from os import getenv
 from pathlib import Path
 
-MODEL_DIR = Path(__file__).parent.parent
+from dotenv import load_dotenv, find_dotenv
+
+load_dotenv(find_dotenv())
+
+MODEL_DIR = Path(__file__).parent.parent if getenv("CACHE_DIR") is None else Path(getenv("CACHE_DIR")) / "cnnensemble"
```

Comment:
In general you want your variable names in code to match the environment variable names so that it is easier to reason about.  In this case I might (1) change to `ZAMBA_CACHE_DIR` to make the env var more explicit, and then (2) have a line above that is `ZAMBA_CACHE_DIR = getenv("ZAMBA_CACHE_DIR", None)`

Reply:
ill want to make sure to change the zamba-web dockerfile too